api/acl.go

```diff
@@ -18,15 +18,14 @@ const (
 	ACLManagementType = "management"
 )
 
-type ACLTokenPolicyLink struct {
-	ID   string
-	Name string
-}
-type ACLTokenRoleLink struct {
+type ACLLink struct {
 	ID   string
 	Name string
 }
 
+type ACLTokenPolicyLink ACLLink
```

Comment:
This feels backwards compatible, but I have a nagging gut feeling like there's some case where this is breaking.

Reply:
The structs have identical fields and the type aliasing gives them their own type names so that none of the rest of the code needed to change. If this somehow isn't backwards compatible then the Go type system is royally messed up.

Reply:
I thought possibly the sharing of the underlying struct definition would somehow show up in `reflect` but I think I poked around enough to quash that nagging feeling.

Reply:
Anyone using those type names can still do so. (De)serialization should be unaffected as the type definitions are identical to what they used to be.

For all intents and purposes these struct definitions were duplicated.